### DISCUSSION TITLE
About "content" in our Tailwind CSS Configuration

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#12662] [Help]
**What version of Tailwind CSS are you using?**

For example: v2.0.4

**What build tool (or framework if it abstracts the build tool) are you using?**

For example: postcss-cli 8.3.1, Next.js 10.0.9, webpack 5.28.0

**What version of Node.js are you using?**

For example: v12.0.0

**What browser are you using?**

For example: Chrome, Safari, or N/A

**What operating system are you using?**

For example: macOS, Windows

**Reproduction URL**

A Tailwind Play link or public GitHub repo that includes a minimal reproduction of the bug. **Please do not link to your actual project**, what we need instead is a _minimal_ reproduction in a fresh project without any unnecessary code. This means it doesn't matter if your real project is private/confidential, since we want a link to a separate, isolated reproduction anyways.

A reproduction is **required** when filing an issue — any issue opened without a reproduction will be closed and you'll be asked to create a new issue that includes a reproduction. We're a small team and we can't keep up with the volume of issues we receive if we need to reproduce each issue from scratch ourselves.

**Describe your issue**

Describe the problem you're seeing, any important steps to reproduce and what behavior you expect instead.

**Comment by ghost** [comment]
**the version of my Tailwind CSS is **tailwindcss@3.4.0**
i am using tailwind css in ReactJS
the version of my NodeJS is **10.2.5****
my browser is Google Chrome and OS is Windows**


Greetings, i am trying to start my server by using npm run dev in vs code but everytime i run that command a warning shows that " No utility classes were detected in your source files. If this is unexpected, double-check the content option in your Tailwind CSS configuration" and i checked the content again and again there is no mistake because i have copied the content directly through Tailwind CSS website please try to help me out in this matter thank you in advance

**Comment by wongjn** [comment]
Consider checking:

- You have used at least one utility class in a file covered by the `content` file globs.
- The `content` file globs you copied from the Tailwind website are relevant to your project.

**Reply by wongjn** [reply]
`'./**/*.{html,js}'` and `'./**/**/*.{html,js}'` are functionally identical. However, the latter glob would include the entire `node_modules` folder which is something you most likely do not want.

**Reply by om-learn** [reply]
omg I deleted my question cause I found something I would consider an answer D: so i didnt want to bother you, but you answered before I could realize, im very sorry.

Edit: and thanks for the answer! I wont use the latter then, as I have no idea what that would do. thanks again!
